diagnostic.md

```diff
@@ -57,13 +63,15 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+li.hide()
 <!-- your answer ends here -->
 
 ## Question 8
 
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
-
+$('#div').on('click', function() {
+  console.log('Good job.')
+}
```

Comment:
You were super close here but `div` is not an ID so you wouldn't need the `#` in front of it. just `'div'` would work.